manifests/Snort/Snort/2.9.16.yaml

```
@@ -0,0 +1,17 @@
+Id: Snort.Snort
+Publisher: Snort
+Name: Snort
+Author: Snort
+Version: 2.9.16
+License: Snort License
```

```suggestion
License: Snort License/GPLv2
```
as per https://www.snort.org/license

---

Thanks! I will change it as soon as possible.